app/controllers/api/v1/confirmations_controller.rb

```diff
@@ -2,6 +2,10 @@
 
 module Api
   module V1
-    class ConfirmationsController < Devise::ConfirmationsController; end
+    class ConfirmationsController < Devise::ConfirmationsController
+      def after_confirmation_path_for(_resource_name, _resource)
+        success_messages_path
```

Comment:
I'm not sure, but should we use the path that generate a new token? This way we can get both a success http status AND the token allowing to start using the API directly? 

(and the bonus is that then we do not need an empty "SuccessMessageController" 🙈 )

Reply:
Usually, when I have to verify my account, I tap on the link in a confirmation email, and it lands me on a web page with a success message. Then I must log in with my email and password to continue using the application. 

I will create another story for your suggestion and work on that.

Reply:
As you prefer ;-) Just an optional suggestion.

From my experience, I'm most of the time already logged in to the service when I confirm my account, so I land on the already logged in interface :harold:

Reply:
@phongvhd93 I think we can do the minimal (I also like the login via confirmation) but up to your preference 🙏 .